src/main/scala/dissonance/DiscordClient.scala

```diff
@@ -39,9 +45,137 @@ class DiscordClient(token: String, client: Client[IO]) {
           headers(token)
         )
       )
+
+  def createWebhook(name: String, avatar: Option[ImageDataUri], channelId: Snowflake): IO[Webhook] =
+    client
+      .expect[Webhook](
+        POST(
+          // TODO Case class here
```

Comment:
I had these TODOs here as well, but now I'm thinking we probably just keep it this way. We already have a ton of models and I don't know if we want more for stuff like this that is only used once. What do you think?

Reply:
I'm also wondering how we're going to be able to write tests for this stuff. Do we want to have integration tests that run against a test guild?